### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by ashwanthdurairaj**
Create week 3 assignment for review

### CODE REVIEW

File: week3/src/App.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import {useState} from 'react'
+
+function Square({value, onSquareClick})
```
Comment (DonBrowny) [inline week3/src/App.js]: create a file names `square.jsx` and move the Squarer component code to it.

File: week3/src/App.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+{
+  return (
+  <button className="square" onClick={onSquareClick}>{value}</button>
+  )
+}
+
+function Board({xIsNext, squares, onPlay})
```
Comment (DonBrowny) [inline week3/src/App.js]: create a file names `board.jsx` and move the Squarer component code to it.

File: week3/src/App.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        <Board xIsNext={xIsNext} squares={currentSquares} onPlay={handlePlay}/>
+      </div>
+      <div className="game-info">
+      {moves}
+      </div>
+    </div>
+  )
+}
+
+function calculateWinner(squares) {
```
Comment (DonBrowny) [inline week3/src/App.js]: create a file names `utile/calculate-winner.js` and move the calcaulteWinner function to the new file.

**Review by DonBrowny — CHANGES_REQUESTED** [review]
Convert this project from npm to yarn. Also try adding [prettier](https://yarnpkg.com/package?name=prettier) package to this repo and modify your VS code to use  prettier

**Comment by ashwanthdurairaj** [comment]
converted project from npm to yarn, added prettier to the existing project, moved all components into a separate component folder and functions into a separate utils folder

**Review by DonBrowny — APPROVED** [review]

**Pull request merged**